cocos/tween/tween.ts

```diff
@@ -42,8 +42,6 @@ type OmitType<Base, Type> = KeyPartial<Base, AllowedNames<Base, Type>>;
 type ConstructorType<T> = OmitType<T, Function>;
 type TweenWithNodeTargetOrUnknown<T> = T extends Node ? Tween<T> : unknown;
 
-const notIntervalPrompt = 'the last action is not ActionInterval';
-
```

Comment:
This line was added by mistake, it's my staged code of refactoring `repeat` action.